### PR TITLE
MWPW-137345 | [For Stage] Interactive container block for CC only

### PR DESCRIPTION
**Pull request by drashti1712**
* Implemented the base Interactive Marquee container for the upcoming variant - Genfill and Firefly
* Figma: https://www.figma.com/file/L1YpIBovlcXffNVvOePgA9/Interactive-Marquee-KR?type=design&node-id=56-1516&mode=design&t=URlCADx9xEuFjjuk-0

Resolves: [MWPW-138595](https://jira.corp.adobe.com/browse/MWPW-138595)

**Test URLs:**
- Before: https://main--cc--adobecom.hlx.page/drafts/drashti/MWPW-137345/container?martech=off
- After: https://stage--cc--drashti1712.hlx.page/drafts/drashti/MWPW-137345/container?martech=off

### CODE REVIEW

**Review by aishwaryamathuria — APPROVED** [review]

File: creativecloud/blocks/interactive-marquee/interactive-marquee.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import { setLibs } from '../../scripts/utils.js';
+
+const miloLibs = setLibs('/libs');
+
+const { decorateButtons, decorateBlockBg } = await import(`${miloLibs}/utils/decorate.js`);
+const { createTag, loadStyle } = await import(`${miloLibs}/utils/utils.js`);
+
+// [headingSize, bodySize, detailSize, titlesize]
```
Comment (salonijain3) [inline creativecloud/blocks/interactive-marquee/interactive-marquee.js:8]: remove comments

**Review by salonijain3 — APPROVED** [review]

**Comment by suhjainadobe** [comment]
The init function can be changed afterwards once https://github.com/adobecom/cc/pull/108/files to incorporate switch case.

**Review by suhjainadobe — APPROVED** [review]

File: creativecloud/blocks/interactive-marquee/interactive-marquee.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  decorate(heading, config);
+}
+
+function extendButtonsClass(text) {
+  const buttons = text.querySelectorAll('.con-button');
+  if (buttons.length === 0) return;
+  buttons.forEach((button) => { button.classList.add('button-justified-mobile'); });
+}
+
+function interactiveInit(el) {
```
Comment (Ruchika4) [inline creativecloud/blocks/interactive-marquee/interactive-marquee.js:42]: can we make this function async? I have an await call to add in this function to load firefly js file.

**Review by Ruchika4 — CHANGES_REQUESTED** [review]

**Review by Ruchika4 — APPROVED** [review]

**Comment by Ruchika4** [comment]
Merging this PR to raise the PR for firefly marquee

**Pull request merged**